packages/server/src/seed.ts

```diff
@@ -90,6 +94,29 @@ async function createSuperAdmin(systemRepo: Repository): Promise<void> {
     profile: createReference(practitioner),
     admin: true,
   });
+
+  if (config.defaultSuperAdminClientId && config.defaultSuperAdminClientSecret) {
+    // Use specified client ID and secret
+    const client = await systemRepo.updateResource<ClientApplication>({
```

Comment:
Should this be a `createResource` rather than `updateResource`? 🤔 Or does it even matter with `systemRepo`?